CONTRIBUTING.md

```diff
@@ -41,13 +41,23 @@ Once it's approved, click "Squash and merge" to publish your changes. Share the
 
 ## Contributing code
 
+### Need to know
+
+Every contribution must be **design reviewed** and **communicated**.
+
+- **Kaizen Site** changes use `docs: ` at the start of the first commit message and PR title.
+- **Major** component changes use `feat: ` at the start of PR titles. For 1 commit, use `feat: ` in the commit message too.
+- **Minor** component changes use `fix: ` at the start of PR titles. For 1 commit, use `fix: ` in the commit message too.
+- **Breaking changes** that are not backwards compatible use feat or fix as above and include `BREAKING CHANGE: ` in the body of a commit message.
+- **Design token** changes… let's talk about that.
+
 ### Quality and reviews
 
 Every code contribution **must** have:
 
-- A design review from a designer
-- A PR summary that describes the changes and anything others should be aware of
-- Communications: share your new component or changes on Slack or at the Front End Practice meeting
+1. A design review from a designer
+2. Semantic versioning and conventional commit PR titles: see [Releasing packages](#releasing-packages)
+3. Communications: share your new component or changes on Slack and at relevant Front End Practice meetings
 
```

Comment:
This is off topic for the changes in this PR, and we can discuss offline, but I feel like QA could easily be a requirement, particularly because of how easy branch previews and storybook make it to QA. I've seen many bugs get through that could have easily been caught with a less than 5 minute minute QA from another person. The other benefit of getting someone else to QA is that it spreads more awareness of the work that's being done (that's one more person that knows about the new component or the changes)